selenium/src/test/java/au/org/emii/portal/tests/step1/FacetTest.java

```diff
@@ -65,6 +66,68 @@ public void parameterPhysicalAtmosphereTest() throws InterruptedException {
     }
 
     @Test
+    public void untickFacetTest() throws InterruptedException {
+        log.info("Loading search page - Step 1");
+        WebDriver driver = getDriver();
+        driver.get(AODN_PORTAL_SEARCH_PAGE);
+
+
+        // grab the text of the original results so we can make sure it reverts back when we untick later
+        String originalResults = webElementUtil.findElement(By.className("faceted-search-results")).getText();
+
+        WebElement topPanel = webElementUtil.findElement(By.className("search-filter-panel"));
```

Comment:
I feel like that would add extra clutter given the code would be unlikely to be used again outside of this method.